packages/shorebird_code_push_client/lib/src/code_push_client.dart

```diff
@@ -245,16 +246,14 @@ class CodePushClient {
   Future<Patch> createPatch({
     required String appId,
     required int releaseId,
-    required bool hasAssetChanges,
-    required bool hasNativeChanges,
+    required CreatePatchMetadata metadata,
   }) async {
     final request = CreatePatchRequest(
       releaseId: releaseId,
       wasForced: false,
-      hasAssetChanges: hasAssetChanges,
-      hasNativeChanges: hasNativeChanges,
-      // TODO(bryanoltman): add metadata
-      metadata: null,
+      hasAssetChanges: metadata.hasAssetChanges,
+      hasNativeChanges: metadata.hasNativeChanges,
```

Comment:
Why not remove the redundant `hasAssetChanges` and `hasNativeChanges` here since they are part of the metadata?

Reply:
Can do now that the backend change is rolled out. Updating

Reply:
Actually, the server still expects this. Will remove once we roll out a new version of the CLI that includes this PR